src/wallet/wallet.h

```diff
@@ -1299,7 +1303,14 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     std::vector<uint256> ResendWalletTransactionsBefore(int64_t nTime);
     CAmount GetBalance() const;
     CAmount GetUnconfirmedBalance() const;
-    CAmount GetUnconfirmedData(const CScript& scriptToMatch, int& numbOfUnconfirmedTx) const;
+
+    typedef enum {
+        ZCC_FALSE = 0,
+        ZCC_TRUE,
+        ZCC_UNDEF
+    } eZeroConfChangeUsage;
```

Comment:
I should be safer to use an enum class instead of a simple enum. For instance, enum classes forbids implicit constructions from int